tests/packagedcode/data/npm/overrides_test/package.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "my-override-example",
+    "version": "1.0.0",
+    "overrides": {

```

Comment:
Can you add real examples seen in the wild? 
See https://github.com/search?q=path%3A*%2Fpackage.json+overrides&type=code